src/runtime/logger.js

```diff
@@ -20,14 +20,20 @@ function getLogger (params) {
 
   const transports = [
     new winston.transports.Console({ format }),
-    new winston.transports.File({ filename: 'stdout', format })
   ]
 
-  if (params.testOutputsPath) {
+  if (process.title !== 'browser') {
     transports.push(new winston.transports.File({
       format,
-      filename: path.join(params.testOutputsPath, 'run.out')
+      filename: 'stdout'
```

Comment:
I believe this is due to a merge from an older master version. Should I remove this all together?

Reply:
I don't understand, what did you merge? What would you like to remove?

Reply:
not sure anymore, going to clean it up.